src/components/menu-top/MenuTop.vue

```diff
@@ -136,6 +144,28 @@ export default Vue.extend({
       },
     },
   },
+  methods: {
+    saveFile(): void {
+      const filename = this.$store.getters.getShowTitle + '.shw';
+      const jsonData = JSON.stringify(this.$store.getters.getShow);
+      const blob = new Blob([jsonData], { type: 'text/plain;charset=utf-8;' });
+      if (navigator.msSaveBlob) { // IE 10+
+        navigator.msSaveBlob(blob, filename);
+      } else {
+        const link = document.createElement('a');
```

Comment:
So what this is doing is creating an `<a>` element with the `download` attribute equal to the data blob. If you look at the docs for a navbar item (https://buefy.org/documentation/navbar), you'll see that the `tag` attribute is by default `a`. This means that `<b-navbar-item>` turns into `<a>`.

This means that we can directly add attributes to `<b-navbar-item>` that we would give to `<a>`. So something like:
```vue
<b-navbar-item
  data-test="menu-top--save-show"
  :href="showFilename"
  :download="showObjectUrl"
>
  Save Show
</b-navbar-item>
```

And in `computed`, you could add fields for the filename and object url like:

```ts
showFilename(): string {
  return this.$store.getters.getShowTitle + '.shw';
},
showObjectUrl(): string {
  const jsonData = JSON.stringify(this.$store.getters.getShow);
  const blob = new Blob([jsonData], { type: 'text/plain;charset=utf-8;' });
  return URL.createObjectURL(blob);
},
```

Checkout the mozilla dev docs for the `<a>` element: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a

Reply:
This makes a ton of sense! Thank you

Reply:
I made the changes as recommended but I can't get the herf attribute to show up on the a tag, but the download attribute correctly populates.
![image](https://user-images.githubusercontent.com/29964426/75834425-c3262a00-5d70-11ea-9b6e-838ba166521f.png)